src/ui/components/atoms/label-icon/styles.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../../../globals/variables';
+@import '../../../../globals/mixins';
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+}
+
+.label {
+  color: $ads-color-main;
+  font-family: Roboto, sans-serif;
+  font-size: $ads-size-main;
+  font-weight: $ads-font-weight-second;
+  margin-top: 0.5rem;
+  text-align: center;
+
+  @include respond-to('medium') {
+    font-size: 1.25rem;
+    line-height: 1.465rem;
+  }
+}
+
+.icon {
+  align-items: center;
+  border-radius: 100%;
+  content: '';
+  display: flex;
+  height: 2.563rem;
+  justify-content: center;
+  margin-bottom: 0.001rem;
+  width: 2.563rem;
+
+  &--primary {
+    background-color: $ads-warning-main;
```

Comment:
Here I like how the variants are being used, their use is correct but the color should be the default color